experiments/integrated/fluxnet/run_fluxnet.jl

```diff
@@ -29,580 +29,322 @@ include(joinpath(climaland_dir, "experiments/integrated/fluxnet/data_tools.jl"))
 include(joinpath(climaland_dir, "experiments/integrated/fluxnet/plot_utils.jl"))
 
 # Read in the site to be run from the command line
-if length(ARGS) < 1
-    error("Must provide site ID on command line")
+#if length(ARGS) < 1
+#    error("Must provide site ID on command line")
+#end
+#
```

Comment:
Is this commented out on purpose?

Reply:
It would be good to continue supporting the previous behavior (i.e. running from the command line). So it would be good to check that at least 2 ARGS are passed in, since now we need `site_ID` and `params`